server/graphql/v2/object/AccountStats.js

```diff
@@ -376,17 +380,60 @@ export const GraphQLAccountStats = new GraphQLObjectType({
           if (!['monthly', 'yearly'].includes(key)) {
             throw new Error('Unsupported frequency.');
           }
-          const stats = await req.loaders.Collective.stats.activeRecurringContributions.load(collective.id);
-          const currency = collective.currency;
-          // There is no guarantee that stats are returned in collective.currency, we convert to be sure
-          const fxRate = await getFxRate(stats.currency, currency);
-          const value = Math.round(stats[key] * fxRate);
+          const loader = req.loaders.Collective.stats.activeRecurringContributions.buildLoader({
+            currency: collective.currency,
+          });
+          const stats = await loader.load(collective.id);
           return {
-            value: value,
-            currency: currency,
+            value: stats[key],
+            currency: collective.currency,
           };
         },
       },
+      activeRecurringContributionsBreakdown: {
+        description: 'Returns some statistics about active recurring contributions, broken down by frequency',
+        type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLAmountStats))),
+        args: {
+          frequency: {
+            type: GraphQLContributionFrequency,
+            description: 'Return only the stats for this frequency',
+          },
+          hasPortability: {
+            type: GraphQLBoolean,
+            description: 'Filter contributions on whether they can be ported to another fiscal host directly',
+          },
```

Comment:
Are we gonna use that concept in the end?

Reply:
Not currently. If we're settling with the fact that recurring contributions can never be transferred directly, we should remove it. I'm adding a reminder to myself to go back and remove this, depending on how things evolved with the current situation.